packages/launcher/test/unit/detect_spec.ts

```diff
@@ -1,6 +1,6 @@
 const detect = require('../../lib/detect').default
-const log = require('debug')('cypress:launcher:test')
 const os = require('os')
+import { log } from '../log'
```

Comment:
ohh, my bad, yes you have replaced `const log = ` with `import { log }` which makes sense.